packages/frontend/src/components/MkEmojiPicker.vue

```diff
@@ -470,6 +472,15 @@ defineExpose({
 						min-width: 0;
 					}
 				}
+
+				> .body.reactionPickerExpandWide {
+					display: flex;
```

Comment:
This overrides `display: grid` of `asDrawer` and thus ignores `reactionPickerWidth` option, I wonder how could that be fixed 🤔